TUTORIAL.md

```diff
@@ -257,7 +257,16 @@ See  Help in the **Snippet Generator** for the `withEnv` step for more details o
 
 ## Build Parameters
 
-If you have configured your pipeline to accept parameters when it is built — **Build with Parameters** — they are accessible as Groovy variables of the same name.
+If you have configured your pipeline to accept parameters when it is built — **Build with Parameters** — they are accessible as Groovy variables inside `params`. Example:
```

Comment:
Well, they are _also_ accessible as environment variables (and thus looking like Groovy variables) of the same name, though as plain text and not considering default values. `params` is better for most purposes so I am fine with just mentioning it.